src/video/uikit/SDL_uikitmodes.m

```diff
@@ -247,6 +273,42 @@ int UIKit_AddDisplay(UIScreen *uiscreen, SDL_bool send_event)
     }
     return 0;
 }
+#endif
+
+#if TARGET_OS_XR
+int UIKit_AddDisplay(SDL_bool send_event){
+    //FIXME: don't hardcode display information
+    CGSize size = CGSizeMake(1920, 1080);
```

Comment:
Can you please fix this FIXME?
Maybe make the resolution a hint, or at least define it in SDL_uikitmodes.h?